pkg/apis/contrail/v1alpha1/kubemanager_types.go

```diff
@@ -54,32 +55,29 @@ type KubemanagerStatus struct {
-	KubernetesAPIServer   string                          `json:"kubernetesAPIServer,omitempty"`
-	KubernetesAPIPort     *int                            `json:"kubernetesAPIPort,omitempty"`
-	KubernetesAPISSLPort  *int                            `json:"kubernetesAPISSLPort,omitempty"`
-	PodSubnets            string                          `json:"podSubnets,omitempty"`
-	ServiceSubnets        string                          `json:"serviceSubnets,omitempty"`
-	KubernetesClusterName string                          `json:"kubernetesClusterName,omitempty"`
-	IPFabricSubnets       string                          `json:"ipFabricSubnets,omitempty"`
-	IPFabricForwarding    *bool                           `json:"ipFabricForwarding,omitempty"`
-	IPFabricSnat          *bool                           `json:"ipFabricSnat,omitempty"`
-	KubernetesTokenFile   string                          `json:"kubernetesTokenFile,omitempty"`
-	HostNetworkService    *bool                           `json:"hostNetworkService,omitempty"`
-	RabbitmqUser          string                          `json:"rabbitmqUser,omitempty"`
-	RabbitmqPassword      string                          `json:"rabbitmqPassword,omitempty"`
-	RabbitmqVhost         string                          `json:"rabbitmqVhost,omitempty"`
-	StaticConfiguration   *KubemanagerStaticConfiguration `json:"staticConfiguration,omitempty"`
+	Containers            []*Container `json:"containers,omitempty"`
+	UseKubeadmConfig      *bool        `json:"useKubeadmConfig,omitempty"`
+	ServiceAccount        string       `json:"serviceAccount,omitempty"`
+	ClusterRole           string       `json:"clusterRole,omitempty"`
+	ClusterRoleBinding    string       `json:"clusterRoleBinding,omitempty"`
+	CloudOrchestrator     string       `json:"cloudOrchestrator,omitempty"`
+	KubernetesAPIServer   string       `json:"kubernetesAPIServer,omitempty"`
+	KubernetesAPIPort     *int         `json:"kubernetesAPIPort,omitempty"`
+	KubernetesAPISSLPort  *int         `json:"kubernetesAPISSLPort,omitempty"`
+	PodSubnets            string       `json:"podSubnets,omitempty"`
+	ServiceSubnets        string       `json:"serviceSubnets,omitempty"`
+	KubernetesClusterName string       `json:"kubernetesClusterName,omitempty"`
+	IPFabricSubnets       string       `json:"ipFabricSubnets,omitempty"`
+	IPFabricForwarding    *bool        `json:"ipFabricForwarding,omitempty"`
+	IPFabricSnat          *bool        `json:"ipFabricSnat,omitempty"`
+	KubernetesTokenFile   string       `json:"kubernetesTokenFile,omitempty"`
+	HostNetworkService    *bool        `json:"hostNetworkService,omitempty"`
+	RabbitmqUser          string       `json:"rabbitmqUser,omitempty"`
+	RabbitmqPassword      string       `json:"rabbitmqPassword,omitempty"`
+	RabbitmqVhost         string       `json:"rabbitmqVhost,omitempty"`
 }
 
-// KubemanagerStaticConfiguration is the configuration for deployment with static controller.
+// KubemanagerConfiguration is the Spec for the kubemanagers API.
```

Comment:
This comment is about KubemanagerConfiguration however it should be for KubemanagerStaticConfiguration

Reply:
done